config/200-clusterrole.yaml

```diff
@@ -45,6 +42,24 @@ rules:
 ---
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  # This is the access that the controller needs on a per-namespace basis.
+  name: tekton-pipelines-controller-tenant-access
+rules:
+  - apiGroups: [""]
+    resources: ["pods", "pods/log", "namespaces", "secrets", "events", "serviceaccounts", "configmaps", "persistentvolumeclaims", "limitranges"]
```

Comment:
Just noticed this here. Were listing `namespaces` even though they are not a namespaces resource. So we can drop it form here. This is already covered in the `tekton-pipelines-controller-cluster-access` `ClusterRole` anyway. 
This shouldn't cause any issues and should be treated as a no-op, but it would be nice to clean it up.